Cargo.toml

```diff
@@ -13,6 +13,16 @@ homepage = "https://github.com/mongodb/bson-rust"
 documentation = "https://docs.rs/crate/bson"
 edition = "2018"
 
+# By default cargo include everything git include
+# cargo diet can help to manage what's not useful.
+include = [
+    "Cargo.toml",
+    "src/**/*",
```

Comment:
Will this end up adding `src/test` even though it's ignored in `!**/tests/*`? I'm not sure what `include` does in the case of conflicting instructions

Reply:
I was wondering about it it's why I published it as a draft I am investigating it.

Reply:
Oops, sorry, I missed that this was still a draft. Sorry for jumping the gun!

Reply:
I think it might be easier to just use `exclude` instead of `include`: https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields